doc/source/workflow/overview.md

```diff
@@ -0,0 +1,208 @@
+# Overview of Seldon Core Components
+
+
+Seldon Core is build out of many components that deployed to Kubernetes works together as microservices.
+
+
+
+
+These are core Seldon's components:
+- Reusable and non-reusable [model servers](./overview.html#model-servers)
+- [Language Wrappers](./overview.html#language-wrappers) to containerise models
+- [SeldonDeployment](./overview.html#seldondeployment-crd) CRD and [Seldon Core Operator](./overview.html#seldon-core-operator)
+- [Service Orchestrator](./overview.html#service-orchestrator) for advanced inference graphs
+
+as well as integration with third-party systems:
+- Kubernetes Ingress integration with [Ambassador](https://www.getambassador.io/) and [Istio](https://istio.io/)
+- [Metrics](./overview.html#metrics-with-prometheus) with [Prometheus](https://prometheus.io/)
+- [Tracing](./overview.html#distributed-tracing-with-jaeger) with [Jaeger](https://www.jaegertracing.io/)
+- [Endpoint Documentation](./overview.html#endpoint-documentation) with [OpenApi](https://swagger.io/docs/specification/about/)
+
+Keep reading to learn more!
+
+
+![](../images/e2e-model-serving.svg)
+
+
+Usual e2e model serving with Seldon consists of few very important steps:
+1. *Data Scientist* prepare ML `model` using state of the art libraries (mlflow, dvc, xgboost, scikit-learn just to name a few).
+2. Trained model is uploaded to the central repository (e.g. S3 storage).
```

Comment:
This seems to be the flow specifically on prepackaged servers, maybe you could add this after introducing the difference between reusable / non-reusable severs, and show the steps that would be different

Reply:
So basically move it to the reusable model servers and then highlight what changes with the non-reusable ones?

Reply:
Yeah that sounds good, or alternatively you can provide the steps for both to show explicitly what they consist of

Reply:
Could provide both steps steps like:
```
2a. (reusable servers, on graph) ... 
2b. (non-reusable servers) ....
```

Main idea behind these points I had was to describe what is visible on the figure.

Reply:
In updated version (pushing in a moment) - I am stressing that these steps are for `Reusable` Model Servers and in one sentence after describe how the process differs with `Non-Reusable` Model servers. 

I hope that the description in this way will be cleaner and covers both use cases avoid repetition.